src/celengine/render.cpp

```diff
@@ -5334,7 +5336,7 @@ bool Renderer::captureFrame(int x, int y, int w, int h, Renderer::PixelFormat fo
     return glGetError() == GL_NO_ERROR;
 }
 
-void Renderer::drawRectangle(const Rect &r, const Matrix4f &mvp)
+void Renderer::drawRectangle(const Rect &r, const Matrices &mvp)
```

Comment:
please use 2 separate Eigen::Matrix4f instead of Matrices here as well

Reply:
fixed.